egs/tedlium2/asr1/conf/tuning/decode_ngram.yaml

```diff
@@ -0,0 +1,6 @@
+ngram-weight: 1.0
+beam-size: 20
```

Comment:
run.sh for tedliums2 doesn't exist?
Is this a garbage file?

Reply:
done